test/schedule/data_test.exs

```diff
@@ -630,6 +630,23 @@ defmodule Schedule.DataTest do
     end
   end
 
+  describe "stations/1" do
+    test "returns only stations" do
+      [station_1, station_2] = [
+        %Stop{id: "station-1", name: "station 1", location_type: :station},
+        %Stop{id: "station-2", name: "station 2", location_type: :station}
+      ]
+
+      stops = %{
+        station_1.id => station_1,
+        station_2.id => station_2,
+        "stop-1" => build(:gtfs_stop, %{id: "stop-1", location_type: :stop})
```

Comment:
**question:** is there a reason to use the factory here but not for defining the station stops?

Reply:
Good question - there is not! I'll update!